data/successors/successor outfits.txt

```diff
@@ -0,0 +1,635 @@
+
+outfit `"Veusa" Inductive Steering`
+	category "Engines"
+	series "Engines: Successors"
+	index 40
+	"cost" 2925000
+	thumbnail "outfit/large successor steering"
+	licenses
+		"Successor"
+	"mass" 42
+	"outfit space" -42
+	"engine capacity" -42
+	"afterburner energy" -2.4
+	"turn" 2268
+	"turning energy" 3.8
+	"turning heat" 3.1
+	"steering flare sprite" "effect/successor flare/large"
+		"frame rate" 20
+	"steering flare sound" "ion large"
+	description `The "Veusa" series of Successor steering units is significantly smaller than comparable human engines, allowing the Successors to gain some flexibility in the engine layouts of their large ships.`
+
+outfit `"Veusa" Inductive Reverser`
+	category "Engines"
+	series "Engines: Successors"
+	index 42
+	"cost" 3050000
+	thumbnail "outfit/large successor reverse"
+	licenses
+		"Successor"
+	"mass" 35
+	"outfit space" -35
+	"weapon capacity" -35
+	"afterburner energy" -2.1
+	"reverse thrust" 54.75
+	"reverse thrusting energy" 3.7
+	"reverse thrusting heat" 3.0
+	"reverse flare sprite" "effect/successor flare/large"
+		"frame rate" 20
+	"reverse flare sound" "ion large"
+	description "While the invertebrate Successors are relatively unaffected, human pilots must take care when using a reverse thruster this large to avoid dangerous g-force stress on the body."
+	description "	Because a reverse thruster must be facing forwards, it must be installed in the weapon section of your ship instead of the engine section."
+
+
+outfit "Multimodal Armor: Heavy"
+	category "Systems"
+	series "Special Systems: Successors"
+	index 10
+	thumbnail "outfit/successor armor heavy"
+	licenses
+		"High Houses"
+	"hull protection" .20
+	"multimodal armor" -1
+	"unplunderable" 1
+	description "The metamaterial armor of Successor warships can be altered in an appropriately equipped spaceport to change its attributes."
+	description "	This configuration strengthens the bonds between the atoms making up the armor, increasing its resistance to direct damage."
+
+outfit "Multimodal Armor: Keystone"
+	category "Systems"
+	series "Special Systems: Successors"
+	index 40
+	thumbnail "outfit/successor armor keystone"
+	licenses
+		"High Houses"
+	cost 10000000
+	"quantum keystone" 1
+	"scan interference" 3
+	"multimodal armor" -1
+	"unplunderable" 1
+	description "The metamaterial armor of Successor warships can be altered in an appropriately equipped spaceport to change its attributes."
+	description "	This configuration allows a vessel equipped with it to safely pass through unstable wormholes. As a side effect, it also substantially interferes with external scans of a vessel's cargo or outfits."
+
+outfit "Multimodal Armor: Regen"
```

Comment:
I would say this is the strongest of the Multimodal Armor options. Doubling the hull repair of the ship is very powerful - it's even better than gaining a free built-in copy of all the hull repair outfits the ship has equipped, because the energy (and heat, if present) aren't doubled.
With this equipped, the amount of regen per net space from even a Coalition large repair module rises to 2.81, exceeding what most T2 and lower shields provide (for example, the Wanderer Dark Storm at 2.65, or the Bunrodea large shield relay with the maximum number of boosters at 2.5) , and with stronger hull repair outfits (like those of the Bunrodea) the advantage is even greater.
Overall, I think this is probably too strong. I'd suggest reducing the multiplier from +100% to something more like +80% or +75%.